source/documentation/at-rules/forward.html.md.erb

```diff
@@ -1,7 +1,179 @@
 ---
 title: "@forward"
 introduction: >
-  TODO(nweiz): Write this
+  The `@forward` rule makes [mixins](mixin), [functions](function), and
+  [variables](../variables) from another Sass stylesheet available to
+  stylesheets that load your stylesheet with the [`@use` rule](use). It makes it
+  possible to organize Sass libraries across many files, while allowing their
+  users to load a single entrypoint file.
 ---
 
+It's written `@forward "<url>"`. It loads the module at the given URL just like
+`@use`, but it makes the [public][] members of the loaded module available to
+users of your module as though they were defined directly in your module. Those
+members aren't visible in your module, though—if you want that, you'll need to
+write a `@use` rule as well. Don't worry, it'll only load the module once!
+
+[public]: use#private-members
+
+<% fun_fact do %>
+  The `@forward` rule acts just like `@use` when it comes to a module's CSS.
+  Styles from a forwarded module will be included in the compiled CSS output,
+  and your module can [extend][] it, even if it isn't also `@use`d.
```

Comment:
Does this apply to the file in which the `@forward` resides as well? It isn't super clear to me, since the _symbols_ from the forwarded file _aren't_ available.

Reply:
Yes. Clarified.